docs/BR.md

```diff
@@ -921,6 +960,10 @@ If a Random Value is used, the CA SHALL provide a Random Value unique to the cer
   i. 30 days or
   ii. if the Applicant submitted the certificate request, the time frame permitted for reuse of validated information relevant to the certificate (such as in [Section 4.2.1](#421-performing-identification-and-authentication-functions) of this document).
 
+CAs using this method MUST implement Multi-Perspective Issuance Corroboration as specified in Section 3.2.2.9. To count as corroborating, a Network Perspective MUST observe:
+- the same challenge information (i.e. Random Value or Request Token) as the Primary Network Perspective; and
+- evidence confirming the CA's authority to issue as determined by the Primary Network Perspective, as specified in Section 3.2.2.8. 
```

Comment:
```suggestion
```

CAA for IP addresses is not a thing yet (see https://datatracker.ietf.org/doc/draft-chariton-ipcaa/ for a recent effort to define such a mechanism), so I suggested striking any reference to 3.2.2.8 in 3.2.2.5.

Reply:
I think some of the language may have been copied over by accident. I agree it makes sense to strike any reference to 3.2.2.8 here.

Reply:
Bad copy and pasting on my part! Thanks for catching, @CBonnell.